server/app.js

```diff
@@ -62,13 +62,19 @@ let profiles = [
     {userID: 1000, name: 'Mark123', chips: 20000, icon: 'player_icon_1'},
     {userID: 2000, name: 'John456', chips: 40000, icon: 'player_icon_1'},
     {userID: 3000, name: 'Luke854', chips: 60000, icon: 'player_icon_1'},
-    {userID: 4000, name: 'Jane027', chips: 80000, icon: 'player_icon_1'},
-    {userID: 5000, name: 'Stan693', chips: 10000, icon: 'player_icon_1'},
+    {userID: 4000, name: 'Jane027', chips: 180000, icon: 'player_icon_1'},
+    {userID: 5000, name: 'Stan693', chips: 100000, icon: 'player_icon_1'},
     {userID: 6000, name: 'Mack273', chips: 20, icon: 'player_icon_1'},
 ];
 // the entire list of tables, might need a better way to store them?
```

Comment:
I had it set up like so using a JS object. Rooms are all given a unique ID and then stored into the rooms object using their ID as a map.  

let rooms = {};
for (let x = 1; x <= 10; x++) {
        //Create x number of rooms 
        const room = {
            id: uuidv4(),
            name: "Table " + x,
            buyin: 20000,
            blinds: "1000 / 2000",
            sockets: []
        }
        rooms[room.id] = room;
    }

Reply:
Maybe this is a more scale-able approach? 

Need opinions.

Reply:
we definitely need to address this tomorrow.